src/styles/card-content.scss

```diff
@@ -0,0 +1,111 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 17px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+    margin: 16px 0 40px;
+  }
+}
+
+.card-content {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin: 0;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin: 4px 0 17px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 7px;
+
+    &-label {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $main-text-color;
+      text-align: right;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: block;
+    box-sizing: border-box;
+    position: relative;
+    align-self: center;
+    margin: 0;
+    width: 166px;
+    height: 40px;
+    flex-shrink: 0;
+    border-radius: 5px;
+    background-color: #00acdc;
+
+    &-buy {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      text-align: center;
+      text-decoration: none;
+      color: #fff;
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+
+      &:hover {
```

Comment:
to see text I should hover exactly the text:
[Screencast from 27-08-23 10:35:37.webm](https://github.com/mate-academy/layout_product-cards/assets/86923089/39497535-f5a9-4350-8e80-9005c8be1e18)


it should be visible on the button hover

generally, you don't need `button` div wrapper - just apply all styles you need to `a` tag (but of course using the class selector)